src/main/java/org/jvnet/hudson/plugins/port_allocator/PortAllocator.java

```diff
@@ -190,7 +190,7 @@ private void checkPortNumbers(String ports) throws FormException {
 
         public Pool getPoolByName(String poolName) throws PoolNotDefinedException {
             for (Pool p : pools) {
-                if (p.name.toUpperCase().equals(poolName.toUpperCase())) {
+                if (p.name.equals(poolName)) {
```

Comment:
For case-sensitive system i will be unable to use two variables different in registry.